gratipay/utils/i18n.py

```diff
@@ -103,17 +104,19 @@ def get_function_from_rule(rule):
 
 
 def get_text(loc, s, *a, **kw):
+    escape = kw.pop('escape', escape_noop)
```

Comment:
Well, we could avoid having the default in the i18n machinery by ensuring that all the renderers we're using take care of it for us. It does seem like escaping should be baked into renderers.